src/shared_modules/content_manager/tests/unit/CtiDownloader_test.hpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * Wazuh Content Manager - Unit Tests
+ * Copyright (C) 2015, Wazuh Inc.
+ * Dec 01, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef _CTI_DOWNLOADER_TEST_HPP
+#define _CTI_DOWNLOADER_TEST_HPP
+
+#include "fakes/fakeServer.hpp"
+#include "updaterContext.hpp"
+#include "gtest/gtest.h"
+#include <memory>
+
+/**
+ * @brief Runs unit tests for CtiDownloader
+ *
+ */
+class CtiDownloaderTest : public ::testing::Test
+{
+protected:
+    CtiDownloaderTest() = default;
+    ~CtiDownloaderTest() override = default;
+
+    std::shared_ptr<UpdaterContext> m_spUpdaterContext;       ///< UpdaterContext used on the merge pipeline.
```

Comment:
Typo? 

> ///< UpdaterContext used on the **merge** pipeline.

Also present in 
src/shared_modules/content_manager/tests/unit/CtiOffsetDownloader_test.hpp
src/shared_modules/content_manager/tests/unit/CtiSnapshotDownloader_test.hpp

Reply:
Yes, typo. Fixed all of them.